apps/election-manager/src/components/TallyReportSummary.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { Dictionary, Election } from '@votingworks/types'
+
+import * as format from '../utils/format'
+import { getLabelForVotingMethod } from '../utils/votingMethod'
+import { VotingMethod } from '../config/types'
+
+import Table, { TD } from './Table'
+
+const BallotSummary = styled.div`
+  margin-bottom: 1em;
+  border: 1px solid rgb(194, 200, 203);
+  border-width: 0 1px;
+  h3 {
+    margin: 0;
+    background: rgb(194, 200, 203);
+    padding: 0.25rem 0.5rem;
+  }
+`
+
+interface Props {
+  election: Election
+  internalBallotCount: number
+  externalBallotCount?: number
+  ballotCountsByVotingMethod: Dictionary<number>
+}
+
+const TallyReportSummary: React.FC<Props> = ({
+  internalBallotCount,
+  externalBallotCount,
+  ballotCountsByVotingMethod,
+}) => {
+  const totalBallotCount = internalBallotCount + (externalBallotCount ?? 0)
+
+  return (
+    <BallotSummary>
+      <h3>Ballots by Voting Method</h3>
+      <Table data-testid="voting-method-table">
+        <tbody>
+          {Object.keys(ballotCountsByVotingMethod).map((votingMethod) => {
```

Comment:
This feels like it's begging to be a `Map` or an array of objects.

Reply:
Would like to refactor but it's outside the scope of this PR. 🙃